test/BDNPerfTests/ConfigFiles/CI_CONFIG_BDN_Benchmark_Cluster.ClusterOperations.json

```diff
@@ -0,0 +1,31 @@
+{
+    "configuration": "Release",
+    "framework": "net8.0",
+    "filter": "BDN.benchmark.Cluster.ClusterOperations.*",
+
+    "expectedGet_DSV_AllocatedValue_linux": 0,
```

Comment:
i think we should only have to add files and entries non-zero allocated. the script should assume, if it does not find a benchmark entry, that the allocated should be zero. that way this entire file gets deleted. and, many lines in other files get deleted as well. will make maintenance a tad easier.

Reply:
I see where you are coming from but not sure I agree with this.  Maintenance is easiest if it is consistent so we know one config file per BDN run. Also, assuming it should be zero if not there is an assumption and it might be better to specifically state to show we know it should be zero.  Currently, the BDN name/ filter is in the config file as well, so that would be a change to only just use the test name.

Reply:
Since we are only pulling values we need for the specific test (based on other comment), we need to know what values we need for that specific test. Those specific values are in the config file, so I think it only makes sense to have all config files even if the values are 0.

Reply:
we could have one config file that has a section per test. for every test we add we would simply run the test, and for every result, we would comare it to either zero (by default) or an override (if specified in the JSON).

Reply:
Yeah, I guess that would be another way to implement it

Reply:
I will try to implement it with one config file for all BDN test runs

Reply:
This is now implemented in a one config with a section per test.